packages/plugin-ext/src/plugin/editors-and-documents.ts

```diff
@@ -129,12 +130,16 @@ export class EditorsAndDocumentsExtImpl implements EditorsAndDocumentsExt {
         }
 
         if (delta.newActiveEditor !== undefined) {
-            this._onDidChangeActiveTextEditors.fire(this.activeEditor());
+            ok(delta.newActiveEditor === null || this.editors.has(delta.newActiveEditor), `active editor '${delta.newActiveEditor}' doesn't exist`);
```

Comment:
What `ok` does here?

Reply:
It's like assert, checks correctness of data.

Reply:
ok